zend_abstract_interface/json/json.c

```diff
@@ -22,16 +22,32 @@ __attribute__((weak)) int php_json_encode(smart_str *buf, zval *val, int options
 __attribute__((weak)) int php_json_decode_ex(zval *return_value, const char *str, size_t str_len, zend_long options,
                                              zend_long depth);
 #endif
+__attribute__((weak)) zend_class_entry *php_json_serializable_ce;
+
+void zai_fetch_json_serializable_ce(zend_module_entry *json_me) {
+    if (php_json_serializable_ce) return;
+
+    if (json_me == NULL) {
+        json_me = zend_hash_str_find_ptr(&module_registry, ZEND_STRL("json"));
+    }
+
+    zend_class_entry **tmp_json_serializable_ce = (zend_class_entry **)DL_FETCH_SYMBOL(json_me->handle, "php_json_serializable_ce");
+    if (tmp_json_serializable_ce == NULL) {
+        tmp_json_serializable_ce = (zend_class_entry **)DL_FETCH_SYMBOL(json_me->handle, "_php_json_serializable_ce");
+    }
+    if (tmp_json_serializable_ce != NULL) {
+        php_json_serializable_ce = *tmp_json_serializable_ce;
+    }
+}
```

Comment:
This is ... dead code, right? I see no usages of that.

Reply:
yep